static/src/javascripts/projects/common/modules/identity/modules/modal.js

```diff
@@ -57,4 +58,4 @@ const hide = (name: string): Promise<void> =>
         })
     );
 
-export { hide, show, getContents };
```

Comment:
nope! this is intentionally left unexported to discourage messing with the modal itself!